tests/attestation_plugin_cert/README.md

```diff
@@ -0,0 +1,19 @@
+Certificate based attestation: plugin model
+=====================
+
+This test validates the following OE public APIs:
+
+* oe_generate_plugin_attestation_certificate
+* oe_free_attestation_certificate, and
+* oe_verify_attestation_certificate
+
+Test scenario:
+
+- **Host Side**
+  1. Create an enclave
+  2. Issue an ecall (get_tls_cert) into enclave for getting a self-signed certificate embedded with a plugin attestation report of the enclave
```

Comment:
The term "report" is SGX specific.  Is this meant to be SGX specific?

For example, in some evidence formats the evidence is already a certificate (and thus is usable with TLS).  It sounds like the "embedded" point might be SGX-specific?

Also, some authentication protocols use JWTs or CWTs or instead of certificates per se.  Would this require we have separate public APIs for certificates, JWTs, CWTs, etc.?

Reply:
I replaced `report` with `evidence` to be TEE agnostic in this file. These two functions `oe_get_attestation_certificate_with_evidence` and `oe_verify_attestation_certificate_with_evidence`  could be applicable to non-SGX TEEs, but the plugins/formats we have so far are SGX specific.